source/open_source/index.html

```diff
@@ -41,7 +41,9 @@ <h2>Beadledom</h2>
           <h2>Bunsen</h2>
```

Comment:
Bunsen and Canadarm appear to be missing row-fluid div's. I wonder if that's why the vertical alignment isn't a 100% perfect vertically aligned grid?

Reply:
or maybe that should / shouldn't be on every element? I'll admit my div / css knowledge is sub zero, but it seems like the alignment vertically of the OSS software page images is a little wonky

Reply:
good catch. missed that I copied an extra div. fixed:
![FireShot Capture 005 - Engineering Health - localhost](https://user-images.githubusercontent.com/22381097/65550465-3cc60c00-dee5-11e9-866b-42cf30498d92.png)